recipes/feature_engine/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "Feature-engine" %}
+{% set version = "0.5.15" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 51c9a50a51e321543f3c88c25ed28674cc1b0235d16a6952d186e6c53e85e007
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
```

Comment:
Does this not depend on scikit-learn?

Reply:
it does. Scikit-learn, pandas, numpy, scipy and statsmodels, as per:
https://github.com/solegalli/feature_engine/blob/master/requirements.txt

do I need to include all those in the meta.yaml?

Reply:
@scopatz do I need to make the packages part of the meta?